pkg/datasource/columns.go

```diff
@@ -272,6 +272,10 @@ var annotationsTemplates = map[string]map[string]string{
 	"containerImageName": {
 		ColumnsWidthAnnotation: "30",
 	},
+	"containerPid": {
+		ColumnsWidthAnnotation:  "6",
+		ColumnsHiddenAnnotation: "true",
```

Comment:
```suggestion
		TemplateAnnotation:  "pid",
		ColumnsHiddenAnnotation: "true",
```

If not, please add `ColumnsAlignmentAnnotation: string(metadatav1.AlignmentRight)` like other numeric fields.